### PR TITLE
Add Babel e2e test

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**

### CODE REVIEW

File: .github/workflows/e2e-tests.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+      - name: Checkout "babel"
+        uses: actions/checkout@v3
+        with:
+          repository: babel/babel
+          path: babel
+
+      - name: Build
+        run: |
+          yarn install --frozen-lockfile
```
Comment (fisker) [inline .github/workflows/e2e-tests.yml:54]: `yarn install --immutable`?

Comment (nicolo-ribaudo) [reply .github/workflows/e2e-tests.yml:54]: Uh yes, I wonder why Yarn didn't complain about the unknown option :thinking:

Comment (nicolo-ribaudo) [reply .github/workflows/e2e-tests.yml:54]: On nevermind, it warns.

Comment (nicolo-ribaudo) [reply .github/workflows/e2e-tests.yml:54]: 65b7725bbfb262ec343000359ef87f345cfca5f9

**Pull request merged**